Integrations/ProofpointTAP_v2/ProofpointTAP_v2_test.py

```diff
@@ -312,3 +312,417 @@ def test_get_fetch_times():
+            "path": "string",
+        },
+        "platforms": PLATFORMS_OBJECT
+    }
+    EVIDENCE_OBJECT_IDS = {
+        "type": "ids",
+        "display": "string",
+        "time": "string",
+        "malicious": "string",
+        "what": {
+            "name": "string",
+            "signatureId": "integer",
+        },
+        "platforms": PLATFORMS_OBJECT
+    }
+    EVIDENCE_OBJECT_FILE = {
+        "type": "file",
+        "display": "string",
+        "time": "string",
+        "malicious": "string",
+        "what": {
+            "action": "string",
+            "md5": "string",
+            "path": "string",
+            "rule": "string",
+            "sha256": "string",
+            "size": "integer"
+        },
+        "platforms": PLATFORMS_OBJECT
+    }
+    EVIDENCE_OBJECT_DROPPER = {
+        "type": "dropper",
+        "display": "string",
+        "time": "string",
+        "malicious": "string",
+        "what": {
+            "path": "string",
+            "rule": "string",
+            "url": "string"
+        },
+        "platforms": PLATFORMS_OBJECT
+    }
+    EVIDENCE_OBJECT_DNS = {
+        "type": "dns",
+        "display": "string",
+        "time": "string",
+        "malicious": "string",
+        "what": {
+            "host": "string",
+            "cnames": ["string1", "string2"],
+            "ips": ["string1", "string2"],
+            "nameservers": ["string1", "string2"],
+            "nameserversList": ["string1", "string2"]
+        },
+        "platforms": PLATFORMS_OBJECT
+    }
+    EVIDENCE_OBJECT_COOKIE = {
+        "type": "cookie",
+        "display": "string",
+        "time": "string",
+        "malicious": "string",
+        "what": {
+            "action": "string",
+            "domain": "string",
+            "key": "string",
+            "value": "string"
+        },
+        "platforms": PLATFORMS_OBJECT
+    }
+    EVIDENCE_OBJECT_ATTACHMENT = {
+        "type": "attachment",
+        "display": "string",
+        "time": "string",
+        "malicious": "string",
+        "what": {
+            "sha256": "string",
+            "md5": "string",
+            "offset": "integer",
+            "rule": "string",
+            "size": "integer"
+        },
+        "platforms": PLATFORMS_OBJECT
+    }
+    EVIDENCE_LIST = [
+        EVIDENCE_OBJECT_ATTACHMENT,
+        EVIDENCE_OBJECT_COOKIE,
+        EVIDENCE_OBJECT_DNS,
+        EVIDENCE_OBJECT_DROPPER,
+        EVIDENCE_OBJECT_FILE,
+        EVIDENCE_OBJECT_IDS,
+        EVIDENCE_OBJECT_MUTEX,
+        EVIDENCE_OBJECT_NETWORK,
+        EVIDENCE_OBJECT_PROCESS,
+        EVIDENCE_OBJECT_REGISTRY,
+        EVIDENCE_OBJECT_URL,
+    ]
+    REPORT_OBJECT = [{
+        'name': 'string',
+        'scope': 'string',
+        'type': 'string',
+        'id': 'string',
+        'forensics': EVIDENCE_LIST,
+    }]
+
+    REPORT = {
+        'generated': 'string',
+        'reports': REPORT_OBJECT * 2,
+    }
+
+    EXPECTED_OUTPUT_REPORT = {
```

Comment:
you dont call your mock data which is global, EXPECTED*.
You should call it FORENSICS_REPORT for example and in the test assign to local variable
`expected_report = FORENSICS_REPORT`